rcv/frame_handlers/frc_2019.py

```diff
@@ -0,0 +1,240 @@
+import cv2
+import math
+import numpy as np
+import os
+import sys
+
+canstants_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(canstants_path)
+from constants import X_PIXELS_PER_DEGREE, Y_PIXELS_PER_DEGREE
+
+response_structure = {
+        "front": False,
+        "found": False,
+        "p1": {"d": 0, "a": 0},
+        "p2": {"d": 0, "a": 0},
+    }
+
+
+class FrameHandler():
+    def __init__(self, configuration):
+        self._conf = configuration
+        self._frame = None
+        self._cleaned_frame = None
+        self._line_contour = None
+        self._points = None
+        self._line_contour = None
+        self._y_leaning_angle = None
+        self._x_turning_angle = None
+        self._response = None
+        self._show = bool(self._conf.get('rcv_server', 'show'))
+        focus_image_path = self._conf.get('rcv_server', 'focus_image')
+        if os.path.exists(focus_image_path) and os.path.isfile(focus_image_path):
+            # TODO do we really need the colored focus image
```

Comment:
we need to show the result of the search, in the variable output in the function show_images, output is in color. And I can't use bitwise and on them if one is grayscale and the other in color so i need the focus image once in grayscale and once in color